datainterface/users.jsx

```diff
@@ -0,0 +1,30 @@
+const { MongoClient } = require("mongodb");
```

Comment:
File name problem- should end in .js not .jsx

Reply:
changed file name to .js